frontend/src/routes/+layout.svelte

```diff
@@ -0,0 +1,7 @@
+<script>
+	import '../app.css';
+</script>
+
+<div class="relative min-h-screen flex flex-col items-center justify-center">
```

Comment:
Remove the wrapper div with the classNames. Only keep `<slot />` in the root layout.

Reply:
Do check the dependent pages if they have the correct classes when you remove this line.